apps/Minecraft Java MultiMC5/install

```diff
@@ -129,7 +152,7 @@ case "$__os_id" in
                 ;;
         esac
         # install dependencies
-        install_packages build-essential libopenal1 x11-xserver-utils git clang cmake curl zlib1g-dev openjdk-8-jre openjdk-11-jdk openjdk-16-jre qtbase5-dev || error "Failed to install dependencies"
+        install_packages build-essential libopenal1 x11-xserver-utils git clang cmake curl zlib1g-dev openjdk-8-jre openjdk-17-jre openjdk-11-jdk openjdk-16-jre qtbase5-dev || error "Failed to install dependencies"
```

Comment:
Are we sure that the openjdk-17-jre package will be available on Buster?

Reply:
its not, make sure to read what that is under
thats the ubuntu only case
buster gets a tar.gz